build.sbt

```diff
@@ -189,9 +189,11 @@ lazy val core = crossProject(JVMPlatform, JSPlatform)
     // Libraries not yet cross-built for Dotty
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % "2.2.0",
-      "org.typelevel" %%% "cats-laws" % "2.2.0" % "test",
+      ("org.typelevel" %%% "cats-laws" % "2.2.0" % "test")
+        .exclude("org.scalacheck", "scalacheck_2.13"),
```

Comment:
This appears to be the problem: we need to use 2.3.0-M1 here, which uses the correct version of Scalacheck.

Reply:
Yeah, but we need a corresponding cats-effect-laws release too, and I don't want to risk releasing FS2 on milestones of cats or cats-effect (like what happened with Spark recently). I think the latest push fixes though.

Reply:
But why not wait until the Dotty builds propagated through the dependency chain? I think this `exclude` hack is potentially dangerous.

Reply:
We could wait, though I really would like this all to be resolved prior or coincident with Dotty 3.0.0-M1, which should be any day now.